vent/menus/add.py

```diff
@@ -16,7 +16,7 @@ def create(self):
         self.repo.when_value_edited()
 
     def quit(self, *args, **kwargs):
-        self.parentApp.switchForm(None)
+        self.parentApp.switchForm('MAIN')
```

Comment:
so the reason that `add`, `add_options`, and `choose_tools` reset the form to `None` instead of `MAIN` is to clear out the parentApp variables that get set and used between classes, for example:

```
self.parentApp.repo_value['repo'] = self.repo.value
```

The other ones don't use a parentApp variable though, so those are probably an oversight that switching back to `MAIN` is fine.